### PR TITLE
Issue #123 and #115

### PR DESCRIPTION
**Pull request by Florent18570**
I tried to resolve the error of my pull request # 127 which I think was an indentation error.

I also check grammatically the sentences of the chatbox response

### CODE REVIEW

File: inference/statnode.py [hunk 1]
```diff
@@ -1,110 +1,215 @@
...
+        for entity in doc.ents:
+            if entity.label_ == "PERSON":
+                name = entity.text
+        if not name:
+            for entity in doc.ents:
+                if entity.label_ == "ORG":
+                    name = entity.text
+
+        return name
+  <<<<<<< Florent18570
```
Comment (skekre98) [inline inference/statnode.py:114]: This line is throwing the error. If you remove it the CI tests should pass :)

**Comment by Florent18570** [comment]
Okay I found out why the check didn't work :)

File: inference/statnode.py (hunk 1 above)
Comment (Florent18570) [reply inference/statnode.py]: Okay I found out why the check didn't work :)

**Review by skekre98 — APPROVED** [review]
Cool, LGTM!

**Pull request merged**